src/build-scripts/gh-installdeps.bash

```diff
@@ -41,6 +39,14 @@ if [[ "$ASWF_ORG" != ""  ]] ; then
         export LLVM_ROOT=$PWD
         popd
     fi
+
+    if [[ "$CXX" == "icpc" || "$CC" == "icc" || "$USE_ICC" != "" ]] ; then
+        sudo mv src/build-scripts/oneAPI.repo /etc/yum.repos.d
```

Comment:
Any reason this is a "mv" instead of a "cp"?

Reply:
oops, no, probably just a cut and paste error